### PR TITLE
refactor: optimize header lean styles

### PR DESCRIPTION
**Pull request by kyubisation**

### CODE REVIEW

File: src/angular/header-lean/header-menu.scss [hunk 1]
```diff
@@ -30,52 +23,50 @@ $sbbHeaderMenuWidth: pxToRem(300);
...
+    color: var(--sbb-color-call-to-action-hover);
   }
 }
 
 .sbb-header-menu > div.sbb-header-menu-title {
-  font-size: pxToRem(21);
-  line-height: pxToRem(25);
-  padding-top: pxToRem(15);
-  padding-left: pxToRem(15);
+  font-size: var(--sbb-header-lean-label-font-size);
```
Comment (jeripeierSBB) [inline src/angular/header-lean/header-menu.scss:51]: more and more I see the font-size 21px, I think it's time to introduce a var?

**Review by jeripeierSBB — CHANGES_REQUESTED** [review]

**Comment by jeripeierSBB** [comment]
yeah, a light header for standard design :-)

File: src/angular/header-lean/header-menu-trigger.scss [hunk 2]
```diff
@@ -48,16 +47,16 @@
 .sbb-header-menu-trigger .sbb-header-menu-trigger-indicator.sbb-icon {
```
Comment (kyubisation) [inline src/angular/header-lean/header-menu-trigger.scss:47]:
```suggestion
.sbb-header-menu-trigger .sbb-header-menu-trigger-indicator {
```

**Review by jeripeierSBB — APPROVED** [review]
LGTM

**Pull request merged**